react/locales/fr.json

```
@@ -34,9 +34,11 @@
   "addresses.addAddress": "Ajouter addresse",
   "addresses.saveAddress": "Sauvegarder addresse",
   "addresses.deleteAddress": "Supprimer addresse",
+  "addresses.notFound": "Vous n'avez aucune adresse enregistré!",
 
   "payments.cardNumber": "Numéro de carte",
   "payments.billingAddress": "Adresse de facturation",
+  "payments.notFound": "Vous n'avez aucun moyen de paiement enregistré!",
```

Exclamation points take a space before in french.
```suggestion
  "payments.notFound": "Vous n'avez aucun moyen de paiement enregistré !",
```

---

D'accord 👍